tests/zfs-tests/include/libtest.shlib

```diff
@@ -2858,28 +2858,31 @@ function block_device_wait
 	fi
 }
 
-#
-# Synchronize all the data in pool
-#
-# $1 pool name
-#
-function sync_pool #pool
```

Comment:
Wow I have no idea.  I am going to close out this PR and start over I did not touch this part of the code at all